h2o-algos/src/test/java/hex/coxph/CoxPHTest.java

```diff
@@ -1,29 +1,31 @@
 package hex.coxph;
 
 import hex.StringPair;
-import org.junit.BeforeClass;
 import org.junit.Test;
+import org.junit.runner.RunWith;
 import water.*;
 import water.fvec.*;
+import water.runner.CloudSize;
+import water.runner.H2ORunner;
 
 import java.util.HashSet;
 import java.util.Set;
 
 import static org.junit.Assert.*;
+import static water.TestUtil.parse_and_track_test_file;
+import static water.TestUtil.parse_test_file;
 
-public class CoxPHTest extends TestUtil {
-
-  @BeforeClass
-  public static void setup() { stall_till_cloudsize(1); }
+@RunWith(H2ORunner.class)
+@CloudSize(1)
+public class CoxPHTest extends Iced<CoxPHTest> {
```

Comment:
does this extend iced because of some inner class? it should not, make inner class static